Assessments/form_journey_with_explainer.md

```diff
@@ -0,0 +1,229 @@
+# Assessments
+
+This Journey will fetch the assessment specified by the configured slug from ContentRepo, and run the user through the questions.
+
+It keeps track of the score, and writes the risk category result to the flow results.
+
+It also writes the user's answers to the flow results.
+
+At the end of the assessment, it sends a message with the text of the page configured for that risk category.
+
+## Content fields
+
+This Journey does not write to any contact fields.
+
+## Flow results
+
+* `assessment_start` - writes the slug of the started assessment when the assessment run starts
+* `question_num` - the number of the question being answered
+* `answer` - the answer that the user chose
+* `assessment_end` - writes the slug of the assessment when the assessment run ends
+* `assessment_score` - the final score that the user got for the assessment
+* `assessment_risk` - one of low, medium, or high. The final categorised risk for the user's score
+
+## Connections to other Journeys
+
+This Journey does not link to any other Journeys
+
+```stack
+trigger(on: "MESSAGE RECEIVED") when has_only_phrase(event.message.text.body, "forms")
+
+```
+
+<!--
+ dictionary: "config"
+version: "0.1.0"
+columns: [] 
+-->
+
+| Key             | Value                                    |
+| --------------- | ---------------------------------------- |
+| api_token       | 22bbdd2a426526b55df8b3ed77eaa3523acfc6e7 |

```

Comment:
Please remove the API Key. I need to revoke it for Onboarding as well :sweat_smile:

Reply:
I flagged this, but looks like I made the mistake too. Doh!

Reply:
Ready for another lok @HawkiesZA